### ISSUE TITLE
Replace every EXV_ symbol with EXIV2_

### ISSUE THREAD

**Issue by clanmills** [Exiv2/exiv2#1514]
I've never understood the advantage in removing vowels from symbol names.  It's a relic of UNIX in 1968 or something.  It's difficult to type and makes no sense.

Doing this has an enormous ripple through the code base.  Much too brutal to be applied to a release candidate. It'll be done on 'main' early in the v1.00 project.

The only relic of this that should endure is the .exv file format which is supported by ExifTool.

**Comment by kmilos** [comment]
Yeah, it looks like there is no real need to duplicate EXIV2_* cmake options with new EXV_* variables that are used in `cmake/config.h.cmake`, one could use directly the EXIV2_* options in that header template instead; see the example at the bottom [here](https://cmake.org/cmake/help/latest/command/configure_file.html).
